glide-core/src/client/reconnecting_connection.rs

```diff
@@ -165,7 +165,7 @@ impl ReconnectingConnection {
         create_connection(backend, connection_retry_strategy, push_sender).await
     }
 
-    fn node_address(&self) -> String {
+    pub(crate) fn node_address(&self) -> String {
```

Comment:
Why `pub(crate)`?  This module is not re-exported from the client mod.rs file anyway, so it'd never be public to the user.

Reply:
even if it is, isnt it better than just pub?

Reply:
Maybe?  I'm not used to seeing it in most crates I've worked with.  Same with `pub(super)`.  It's not that common and there was a time when neither was supported in the language, so it shouldn't be that necessary.

Reply:
I think that it doesnt really matter

Reply:
Sure, we can leave it alone.  We're already using it everywhere anyway.  Just looks weirdly unidiomatic when compared against every other crate in the ecosystem is all.